templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -175,6 +175,7 @@ contents:
         echo "OVS successfully configured"
         ip a show br-ex
         configure_driver_options ${iface}
+        mv /etc/keepalived/keepalived.conf /etc/keepalived/keepalived.conf.bak
```

Comment:
I think this will fail on platforms that don't use keepalived.

Reply:
> > What would help here I think is to teach NM to have a way to make _transient_ changes to the network setup i.e. don't persist into `/etc`. Then we'd run `configure-ovs.sh` at each boot, and wouldn't need a cleanup script.
> 
> @bcrochet was working on a way to make transient network config changes that might be relevant.
> 
> One concern with the keepalived workaround inline. Ideally we would remove that once my fix is in anyway, but if it unblocks things in the meantime I guess it's fine.

Proposal here: openshift/enhancements#399
PR here: #2017

Reply:
> I think this will fail on platforms that don't use keepalived.

fixed.